CHANGELOG.rst

```diff
@@ -2,7 +2,7 @@ Changelog
 =========
 
 0.1.2 & 0.1.3 (2018-09-10)
-------------------
+--------------------------
```

Comment:
The docs require exact line formatting for these text constructs?

Reply:
yup

Reply:
(and I'm not going to make yet another release, but at least it'll be clean in the repo)